server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -103,6 +103,8 @@ import { ExecuteBash, ExecuteBashOutput, ExecuteBashParams } from './tools/execu
 import { ExplanatoryParams, InvokeOutput, ToolApprovalException } from './tools/toolShared'
 import { ModelServiceException } from './errors'
 import { FileSearch, FileSearchParams } from './tools/fileSearch'
+import { partialClone } from '@aws/lsp-core/out/util/collectionUtils'
```

Comment:
do these need to reference the out folder? can't we just do:
```
import { partialClone, formatObjectForLogs } from '@aws/lsp-core`
```
?

Reply:
nope, this was autofilled. Should have double checked.